pkg/ddl/ddl_api.go

```diff
@@ -3290,11 +3341,13 @@ func (d *ddl) CreateView(ctx sessionctx.Context, s *ast.CreateViewStmt) (err err
 		onExist = OnExistReplace
 	}
 
-	return d.CreateTableWithInfo(ctx, s.ViewName.Schema, tbInfo, onExist)
+	var involvingRef []model.InvolvingSchemaInfo
+	// TODO(lance6716): visit the s.Select to get all TableNames?
```

Comment:
i think it's not needed, view is only a stored sql, it's not checked when operating underly table